packs/feats/echolocation.json

```diff
@@ -31,7 +31,50 @@
             "remaster": true,
             "title": "Pathfinder Howl of the Wild"
         },
-        "rules": [],
+        "rules": [
+            {
+                "itemType": "action",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    {
+                        "or": [
+                            "item:slug:seek",
+                            "item:slug:search"
+                        ]
+                    }
+                ],
+                "property": "description",
+                "value": [
+                    {
+                        "divider": true,
+                        "text": "PF2E.SpecificRule.Werecreature.Echolocation.Note"
+                    }
+                ]
+            },
+            {
+                "acuity": "precise",
+                "key": "Sense",
+                "predicate": [
+                    "echolocation"
+                ],
+                "range": 40,
+                "selector": "echolocation"
+            },
+            {
+                "key": "RollOption",
+                "option": "echolocation",
+                "predicate": [
+                    {
+                        "or": [
+                            "change-shape:hybrid",
+                            "change-shape:animal"
+                        ]
+                    }
+                ],
+                "toggleable": true
+            }
```

Comment:
Since this is a timed effect it should go on an effect. The alteration can add the link to the effect.

Reply:
Should the effect be limited until the end of the current turn (Seek) or unlimited? Because Search can be anything including an exploration activity.

Reply:
I think end of turn. Most groups don't advance time outside of a fight anyways. I'd rather it get it right when it matters